src/chrome/PageControl.tsx

```diff
@@ -29,6 +29,7 @@ const PageControl: React.FC<PageControlProps> = ({
   showRange = true,
   onPageChange
 }) => {
+  pageInfo = pageInfo || {}
```

Comment:
We don't have proper paging responses on core so this fails elsewhere with `pageInfo` being `undefined`. The downside to this fix is you see a bunch of "page 0 out of 0 with 0 results" kind of sections on the UI.

Reply:
interesting. I think we should do two things:
1. get an issue up on core to checklist out all api endpoints that need page info but don't have it
2. this line is still good defensive coding, but can be refactored as a default value in the arguments list:

```tsx
const PageControl: React.FC<PageControlProps> = ({
  pageInfo = {},
}) => {}
```